docs/zoneddatetime.md

```diff
@@ -1200,7 +1200,7 @@ zdt.toMonthDay(); // => 12-07
 zdt.toTime(); // => 03:24:30
 ```
 
-### zonedDateTime.**getFields**() : { year: number, month: number, day: number, hour: number, minute: number, second: number, millisecond: number, microsecond: number, nanosecond: number, calendar: object, [propName: string]: unknown }
+### zonedDateTime.**getFields**() : { year: number, month: number, day: number, hour: number, minute: number, second: number, millisecond: number, microsecond: number, nanosecond: number, offsetNanoseconds: number, timeZone: object, calendar: object, [propName: string]: unknown }
```

Comment:
This recently changed to have `offset` be the field and `offsetNanoseconds` be the convenience property, instead of the other way around. LMK if you need me to dig up the GH issue where we made this change, but the current code in #700 should already have this change included.